install_config/install/docker_registry.adoc

```diff
@@ -173,20 +173,101 @@ $ oc logs docker-registry-1-da73t
 
 == File Storage
 
-Tag and image metadata is stored in OpenShift, but the registry owns layer and
-signature data. The registry stores data in a volume that is mounted into the
-registry container at *_/registry_*. You can
-link:../../dev_guide/executing_remote_commands.html[execute a remote command] in
-the registry container to see the contents:
+Tag and image metadata is stored in OpenShift, but the registry stores layer and
+signature data in a volume that is mounted into the registry container at
+*_/registry_*. As `oc exec` does not work on privileged containers, to find the
+correct node housing the container manually SSH into the node housing the
+container and then run `docker exec` on the pod.
```

Comment:
The two "node housing the container"s seem redundant. Suggest:

"As oc exec does not work on privileged containers, to view a registry's contents you must SSH into the node housing the registry pod's container, then run docker exec on the container itself."